controllers/usersignup/usersignup_controller.go

```diff
@@ -840,6 +846,12 @@ func (r *Reconciler) ensureSpace(logger logr.Logger, userSignup *toolchainv1alph
 	return space, true, nil
 }
 
+func (r *Reconciler) isSpaceReady(logger logr.Logger, space *toolchainv1alpha1.Space) bool {
+	logger.Info("Ensuring Space is Ready", "Space", space.Name)
```

Comment:
```suggestion
	logger.Info("Checking whether Space is Ready", "Space", space.Name)
```
Just a suggested rewording since we're not really "ensuring" it, we're just "checking" it

Reply:
done in [5ea85c0](https://github.com/codeready-toolchain/host-operator/pull/670/commits/5ea85c0b2ec4c72293ee59beea0f8b8d3c8af09d)